frb_example/pure_dart/rust/build.rs

```diff
@@ -1,24 +0,0 @@
-use lib_flutter_rust_bridge_codegen::{
```

Comment:
nit: maybe we should at least remain one build.rs in our integration test system to ensure build.rs does not break (when there is no macros)

Reply:
The one in pure_dart_mutli should still work i think. Not sure why i removed that.
I try to add that again

Reply:
added the build script back in for pure_dart_multi